docs/zh-cn/user_doc/backend-deployment.md

```diff
@@ -4,15 +4,15 @@
 
 ## 1、准备工作
 
-请下载最新版本的安装包，下载地址： [码云下载](https://gitee.com/easyscheduler/EasyScheduler/attach_files/)或者[github下载](https://github.com/analysys/EasyScheduler/releases) ，下载escheduler-backend-x.x.x.tar.gz(后端简称escheduler-backend),escheduler-ui-x.x.x.tar.gz(前端简称escheduler-ui)
+请下载最新版本的安装包，下载地址： [码云下载](https://gitee.com/dolphinscheduler/DolphinScheduler/attach_files/)或者[github下载](https://github.com/analysys/EasyScheduler/releases) ，下载escheduler-backend-x.x.x.tar.gz(后端简称escheduler-backend),escheduler-ui-x.x.x.tar.gz(前端简称escheduler-ui)
```

Comment:
Please remove gitee download link, because apache will give us a link when release version

Reply:
ok, i will delete it later